Core/PaymentMethodHelper.php

```diff
@@ -129,6 +136,174 @@ public static function getBic()
         return $aDynvalues['bic'];
     }
 
+    /**
+     * Returns date of birth
+     *
+     * @return string date of birth formated for db (format 'Y-m-d')
+     *
+     * @since 1.2.0
+     */
+    public static function getDbDateOfBirth()
+    {
+        $oSession = Registry::getConfig()->getSession();
+        $aDynvalues = $oSession->getVariable('dynvalue');
+        $sDateOfBirth = $aDynvalues['dateOfBirth'];
+        $oDateOfBirth = DateTime::createFromFormat(Helper::translate('wd_birthdate_format_php_code'), $sDateOfBirth);
+
+        return $oDateOfBirth
+            ? $oDateOfBirth->format(Helper::translate(self::DB_DATE_FORMAT))
+            : self::DEFAULT_DATE_OF_BIRTH;
+    }
+
+    /**
+     * Sets date of birth
+     *
+     * @param string $sDbDateOfBirth formated for db (format 'Y-m-d')
+     *
+     * @since 1.2.0
+     */
+    public static function setDbDateOfBirth($sDbDateOfBirth)
+    {
+        $oSession = Registry::getConfig()->getSession();
+        $aDynvalues = $oSession->getVariable('dynvalue');
+        $aDynvalues['dateOfBirth'] = '';
+
+        if ($sDbDateOfBirth !== self::DEFAULT_DATE_OF_BIRTH) {
+            $oDateOfBirth = DateTime::createFromFormat(self::DB_DATE_FORMAT, $sDbDateOfBirth);
+
+            if ($oDateOfBirth) {
+                $aDynvalues['dateOfBirth'] =
+                    $oDateOfBirth->format(Helper::translate('wd_birthdate_format_php_code'));
+            }
+        }
+
+        $oSession->setVariable('dynvalue', $aDynvalues);
+    }
+
+    /**
+     * Returns true if a valid date of birth is available
+     *
+     * @return bool
+     *
+     * @since 1.2.0
+     */
+    public static function isDateOfBirthSet()
+    {
+        return PaymentMethodHelper::getDbDateOfBirth() !== self::DEFAULT_DATE_OF_BIRTH;
+    }
+
+    /**
+     * Returns true if user is min 18 years old or date of birth is not known
+     *
+     * @return bool
+     *
+     * @since 1.2.0
+     */
+    public static function isUserEighteen()
```

Comment:
Can we make this more generic, e.g.:

```suggestion
    public static function isUserOlderThan($iAge)
```